docs/api-reference.md

```diff
@@ -11,14 +11,15 @@ route: '/reference/api'
 - [`act`](/reference/api#act)
 - [`cleanup`](/reference/api#cleanup)
 - [`addCleanup`](/reference/api#addcleanup)
+- [`removeCleanup`](/reference/api#removecleanup)
 
 ---
 
 ## `renderHook`
 
-```js
+```ts
 function renderHook(
-  callback: function(props?: any): any,
+  callback: (props?: any): any,
```

Comment:
I don't think this syntax is valid.  This matches what you've got below for other callbacks

```suggestion
  callback: (props?: any) => any,
```